src/main/java/edu/ksu/canvas/attendance/services/AttendanceSectionService.java

```diff
@@ -2,6 +2,10 @@
 
 import java.util.List;
 
+import edu.ksu.canvas.attendance.enums.AttendanceType;
+import edu.ksu.canvas.attendance.form.CourseConfigurationForm;
+import org.apache.commons.lang3.Validate;
+import org.apache.commons.lang3.exception.ContextedRuntimeException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.commons.lang3.exception.ContextedRuntimeException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)